src/com/google/enterprise/adaptor/database/UniqueKey.java

```diff
@@ -82,27 +84,35 @@
       def[1] = def[1].trim();
       String name = def[0];
       ColumnType type;
-      if ("int".equals(def[1].toLowerCase())) {
+      if ("int".equals(def[1].toLowerCase(US))) {
```

Comment:
Getting uglier. Maybe switch (def[1].toLowerCase(US)) { case "int": ... }. Or perhaps better, ColumnType.valueOf(def[1].toUpperCase()), with a less helpful error message, or catch the IllegalArgumentException to produce the better one we have now.